doc/changes/changes_1.7.3.md

```diff
@@ -0,0 +1,16 @@
+# Exasol Kafka Connector Extension 1.7.3, released 2024-02-15
```

Comment:
I am sorry, due to slow review you will need to update this before release.

Reply:
No problem. Let's aim to next Monday :). Will update